client/interfaces.go

```diff
@@ -693,12 +693,18 @@ type InstanceFileArgs struct {
 	// User id that owns the file
 	UID int64
 
+	ModifyExistingUID bool
```

Comment:
Shall we use `Set` rather than `Modify` in general in this PR - as its more concise?

Reply:
Also we generally use the naming format of `<Thing><Action>` so would it make more sense for these new vars to be in the form of `UIDSetIfExists`?

Reply:
I considered using `Set`, but it doesn't imply that we're changing something that already exists in the same way that `Modify` does: `X-LXD-modify-perm` vs `X-LXD-set-perm` vs `X-LXD-set-existing-perm`. It lets the header itself be a little more concise. Since it's for backwards compat, the header probaby wouldn't make much sense for somebody just browsing the API docs, so any clarity we can provide there is good. I will try and elaborate in the docs as well. I'm happy to change to `Set` if you'd like.

How about `UIDModifyExisting` (& friends)? I think
```go
if UIDModifyExisting { ... }
```
reads better than
```go
if UIDModifyIfExists { ... }
```